### ISSUE TITLE
Can not add liquidity with pancakeswap

### ISSUE THREAD

**Issue by ghost** [safemoonprotocol/Safemoon.sol#47]
I take the code from Safemoon and give it another name and symbol, compile it, and deploy it with injected web3 with Metamask and it is successful and I can view it on the Bscscan. But i can not add liquidity on Pancakeswap, when i give the smart contact address in it can not find my token to add liquidity. Does anybody know what's wrong?

**Comment by ricardobatdasilva** [comment]
> I take the code from Safemoon and give it another name and symbol, compile it, and deploy it with injected web3 with Metamask and it is successful and I can view it on the Bscscan. But i can not add liquidity on Pancakeswap, when i give the smart contact address in it can not find my token to add liquidity. Does anybody know what's wrong?


managed to solve?

**Comment by cscottyb** [comment]
Any luck solving this issue?

**Comment by ghost** [comment]
Yeah, basically the safemoon contract consists of multiple little contracts and i was compiling one of these little contracts instead of the whole contract with all the little contracts in it.
